### PR TITLE
Add artsy to examples documentation

### PR DESCRIPTION
**Pull request by shilman**
Issue: N/A

## What I did

Add artsy to examples documentation

localhost_8000_examples_

## How to test

if you must:
```
cd docs
yarn install
yarn dev
open http://localhost:8000/examples/
```

### CODE REVIEW

**Comment by orta** [comment]
Perfect

File: docs/pages/examples/_examples.yml [hunk 1]
```diff
@@ -30,6 +30,13 @@ coursera:
   title: Coursera
   description: Coursera UI component library
   demo: https://building.coursera.org/ui/
+artsy:
+  thumbnail: ./thumbnails/artsy.png
+  title: Artsy Force
+  description: Arty's "Force" component library
```
Comment (alloy) [inline docs/pages/examples/_examples.yml]: Small spelling error here, should be Art**s**y

Comment (shilman) [inline docs/pages/examples/_examples.yml]: argh. thanks

Comment (alloy) [inline docs/pages/examples/_examples.yml]: Thanks for featuring us!

Comment (shilman) [inline docs/pages/examples/_examples.yml]: Fixed! Thanks for using and contributing to Storybook!

**Pull request merged**